### PR TITLE
[ci][champagne/1] script to build anyscale BYOD champagne image

### PR DESCRIPTION
**Pull request by can-anyscale**
A script to build champagne (dog-fooding) ray image for SA. 

Buildkite pipeline is created here and verify that it builds: https://buildkite.com/ray-project/release-tests-champagne/builds/5

Next step, I'll hook the pipeline up with OSS CI postmerge job so we can create a champagne image for every new commit in a release branch.

### CODE REVIEW

File: release/ray_release/byod/build.py [hunk 1]
```diff
@@ -22,6 +27,47 @@
 PYTHON_VERSION = "3.8"
 
 
+def build_anyscale_champagne_image(
```
Comment (aslonnie) [inline release/ray_release/byod/build.py:30]: should we have a unit test for this function too?

Comment (can-anyscale) [reply release/ray_release/byod/build.py:30]: totally!

**Comment by can-anyscale** [comment]
Discussed offline, it doesn't work if we generate an image with fixed name, since it might drift the version between head node and worker node when the image with the same name is updated

File: release/ray_release/byod/build.py (hunk 1 above)
Comment (aslonnie) [reply release/ray_release/byod/build.py:30]: maybe remove the word "anyscale"? I do not think it is really related to "anyscale" here.

where does the word "champagne" come from?

**Review by aslonnie — APPROVED** [review]

File: release/ray_release/byod/build.py (hunk 1 above)
Comment (can-anyscale) [reply release/ray_release/byod/build.py:30]: champagne comes from the marketing team ;), as a replacement suggestion for dogfooding

**Review by matthewdeng — APPROVED** [review]
<div><br/>via <a href="https://giphy.com/taylorswift/">Taylor Swift</a> on <a href="https://giphy.com/gifs/taylorswift-music-video-taylor-swift-the-man-TgC8jEirdyKICUVPPs">GIPHY</a></div>

**Pull request merged**